modules/substrate/src/lib.rs

```diff
@@ -96,18 +96,19 @@ pub trait Trait: system::Trait {}
 decl_storage! {
 	trait Store for Module<T: Trait> as Bridge {
 		/// The number of current bridges managed by the module.
-		pub NumBridges get(num_bridges) config(): BridgeId;
+		pub NumBridges get(fn num_bridges) config(): BridgeId;
 
 		/// Maps a bridge id to a bridge struct. Allows a single
 		/// `bridge` module to manage multiple bridges.
-		pub TrackedBridges get(tracked_bridges): map hasher(blake2_256) BridgeId => Option<BridgeInfo<T>>;
+		pub TrackedBridges get(fn tracked_bridges): map hasher(blake2_128_concat) BridgeId => Option<BridgeInfo<T>>;
 	}
 }
 
 decl_module! {
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
 		type Error = Error<T>;
 
+		#[weight = 0] // TODO: update me
```

Comment:
Can you add an issue number to all the todos?

Reply:
The issue I've filed today is about PoA -> Substrate bridge. I was thinking about submitting second issue about Substrate -> Substrate bridge, but looking at the code - 1 of 2 calls there is a noop and I'm not sure about the future of the other one. So I've decided not to spam issues now - I do not know what's the status/future of that part.